excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/AuthorisationCaseFieldParser.java

```diff
@@ -86,27 +82,6 @@
         return parseResults;
     }
 
-    private void validateCaseFields(Map<String, DefinitionSheet> definitionSheets, DefinitionSheet definitionSheet, String caseTypeReference) {
```

Comment:
Why are we removing this here?

Reply:
or are we moving it to `AuthorisationParser`?

Reply:
yes, as it is used in both subclasses